rke/k8s_rke_system_images.go

```diff
@@ -4364,6 +4364,50 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  m("rancher/kubelet-pause:v0.1.6"),
 			Nodelocal:                 m("k8s.gcr.io/k8s-dns-node-cache:1.15.13"),
 		},
+		// Enabled out of band post v2.5.6
```

Comment:
nit: we use this to keep track of when the version was released, for 1.20 this would be `Enabled with 2.5.6` since it's not going out of band

Reply:
:+1: